src/GafferArnold/ArnoldOptions.cpp

```diff
@@ -174,6 +174,8 @@ ArnoldOptions::ArnoldOptions( const std::string &name )
 	options->addChild( new Gaffer::NameValuePlug( "ai:render_device", new IECore::StringData( "CPU" ), false, "renderDevice" ) );
 	options->addChild( new Gaffer::NameValuePlug( "ai:gpu_max_texture_resolution", new IECore::IntData( 0 ), false, "gpuMaxTextureResolution" ) );
 
+	addChild( new Gaffer::StringPlug( "populateGPUCache" ) );
```

Comment:
We avoid making plugs just to hang a UI element off them - they break the separation between non-gui and gui modules, and make for confusion in the node reference. Instead, we can use `layout:customWidget:*` metadata to insert a widget into the UI directly.

Reply:
That commit has shuffled off its mortal coil in the latest push.